### ISSUE TITLE
i18n support.

### ISSUE THREAD

**Issue by Scrambled777** [blissd/fotema#27]
Hi mate, I am interested in translating this app. 

Will it be possible to have `pot` file in future? Sadly, I lack programming skill to generate POT file by myself.

Cheers.

**Comment by blissd** [comment]
Hi Scrambled777. Thanks for your kind offer. I would be very interested in having Fotema translated into other languages. I don't really know anything about `pot` files and translation so I'll take a look and how it works in GNOME and have a go providing the files you need. Is there any documentation you can point me at to get started?

**Comment by Scrambled777** [comment]
Hi again, really appreciate your reply.

I tried to waddle around in meson documentation to generate `pot` file. Maybe this [link](https://mesonbuild.com/Localisation.html) will help.

Or maybe you can look into Weblate integration. They provide `Gratis Weblate` plan for libre projects. Check [here](https://hosted.weblate.org/hosting/).

**Comment by blissd** [comment]
Apparently I also lack the programming skills to generate a POT file. As far as I can tell it looks like Rust isn't well supported by `gettext` which complicates generating POT files. I succeeded in generating a POT file from non-Rust sources, but that isn't very helpful.

I'll think I'll have to find another Rust app that has a POT file and ask how they generated it.

**Comment by Scrambled777** [comment]
@blissd 

Could this [help](https://github.com/justinas/gettext/issues/12). It resulted in this [blog](https://blog.hackeriet.no/rust-and-translation-files/) which talks about rust strings translation.

**Comment by blissd** [comment]
@Scrambled777 Fotema is now ready for localisation. I've added some instructions in [LOCALISATION.md](https://github.com/blissd/fotema/blob/main/LOCALISATION.md). 

I know that `gettext` is the defacto standard for Linux and GNOME apps, but after much reading about how `gettext` works I found I don't really like it. So instead I've used [Fluent](https://projectfluent.org/) which is by Mozilla and is currently being used in the Firefox browser. Hopefully this isn't too controversial of a choice that I've made.
